TypeSystem/Schema/golden/smoke_test_struct.cc

```diff
@@ -34,20 +34,23 @@ struct Empty {
 struct X {
   int32_t x;
 };
+enum class E : uint16_t {};
 struct Y {
-  int32_t x;
+  E e;
 };
+using MyFreakingVariant = Variant<A, X, Y>;
 struct C {
   Empty e;
-  Variant<X, Y> c;
+  MyFreakingVariant c;
 };
+using Variant_B_A_B_C_Empty_E = Variant<A, B, C, Empty>;
```

Comment:
Ok, found out that you use prefix `B_` and suffix `_E`. Trying to find out, why :)